config/config.txt

```diff
@@ -368,8 +368,11 @@ AUTOADMIN_RANK Game Master
 #AUTOADMIN
 
 
-## Uncomment to automatically deadmin players when the game starts.
-#AUTO_DEADMIN_PLAYERS
+## Uncomment to automatically deadmin players whenever they are controlling a mob.
+#AUTO_DEADMIN_ALWAYS
```

Comment:
@tgstation/head-admins is this something that you think would ever see use over the new config option, here or downstream?

Reply:
Who knows re downstreams. 

There's an admin pref for always deadmin, which is the only reason I suggested we keep it in the linked issue report; to retain parity with the pref plus config override paradigm we have going.

I have no particular attachment to an always deadmin config, I only enabled it because I thought it did what this PR adds rather than generating admin tears at the client freeze when readminning.